hw/ip/keymgr_dpe/data/keymgr_dpe.hjson

```diff
@@ -1131,5 +1131,25 @@ countermeasures: [
       ]
     },
 
+    { name: "LOAD_KEY_REGWEN",
```

Comment:
Should we name this different from REGWEN? Because the behavior is different from a classic REGWEN. Even if the bit is 0, you can write the command but the hardware will trigger an invalid command error in this case.

The behavior sounds very reasonable to me but the naming is counterintuitive. I think this may cause more issues down the line. We could e.g. name it LOAD_KEY_INVALID. WDYT @Razer6 ?

Reply:
@vogelpi What about `LOAD_KEY_LOCK`?